### DISCUSSION TITLE
about useQueries

### DISCUSSION THREAD

**Discussion by qu-chen-x** [TanStack/query#3944] [Q&A]
### Describe the bug

As a junior front-end developer, I recently encountered a small problem when learning react query: when reading usequeries on the official website of react query, I saw this sentence as shown in the figure, but I can't quite understand its meaning. Can someone explain it to me? Thank you very much.

### Your minimal, reproducible example

no

### Steps to reproduce

no

### Expected behavior

no

### Screenshots or Videos

![6e06ac24b290dfdbcd7043308f374ba0](https://user-images.githubusercontent.com/73206178/181675675-e62ec5d6-0654-44cd-829a-79061cdc0f87.png)

### Platform

no

### react-query version

no

### TypeScript version

no

**Comment by TkDodo** [comment]
please have a look at the `rules of hooks` page from the react docs:
https://reactjs.org/docs/hooks-rules.html

**Reply by qu-chen-x** [reply]
> please have a look at the `rules of hooks` page from the react docs: https://reactjs.org/docs/hooks-rules.html
thank you very much!

**Comment by kxxoling** [comment]
**Only Call Hooks at the Top Level**, and don’t call Hooks inside loops, conditions, or nested functions. Then you can safely ignore this statement.

[ESlint rule](https://reactjs.org/docs/hooks-rules.html#eslint-plugin) is helpful to avoid writing smelly codes.
